scripts/publish.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -e
+
+timestamp() {
+  date +"%T"
+}
+
+function publish ()
+{
+    echo "getting swagger codegen jar..."
+    wget http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/2.2.3/swagger-codegen-cli-2.2.3.jar -O swagger-codegen-cli.jar
+    java -jar swagger-codegen-cli.jar
+    java -jar swagger-codegen-cli.jar help
+
+    echo "generating Sam Java Client API..."
+    java -jar swagger-codegen-cli.jar generate -l java --input-spec ./src/main/resources/swagger/api-docs.yaml --output generated
+
+    echo "adding plugins file..."
+    cd generated
+    mkdir project
+    echo "addSbtPlugin(\"com.eed3si9n\" % \"sbt-assembly\" % \"0.14.3\")" > project/plugins.sbt
+    sed -i 's/publishArtifact in (Compile, packageDoc) := false/publishTo := Some(Resolver.file(\"file\", new File(\"publish\")))/g' build.sbt
+
+    echo "compiling Java Client API..."
+    sbt assembly
```

Comment:
This could be 1 sbt command:
```
sbt compile assembly publish
```
Also since compile is a dependency of assembly shouldn't it come first?

Reply:
oh - I didn't realize you could just put in one line, thanks
And that's probably true - I can change the order